src/client/timer.c

```diff
@@ -0,0 +1,117 @@
+#include "timer.h"
+#include <SDL_ttf.h>
+#include <SDL.h>
+#include <SDL_image.h>
+#include "game.h"
+
+void CreateTimer(Timer *timer)
+{
+    TTF_Init(); // va tvungen att lägga den här för att initiera Sans
+
+    timer->mCurrentTime = 0;
+    timer->mDeltaConvert = 0;
+    timer->mDeltaTime = 0;
+    timer->mLast10Sek = false;
+    timer->mMinuteTime = 0;
+    timer->mPrevTime = 0;
+    timer->mStarted = true;
+
+    timer->Sans = TTF_OpenFont("res/fonts/OpenSans-Semibold.ttf", 24);
+    if (!timer->Sans)
+    {
+        printf("TTF_OpenFont: %s\n", TTF_GetError());
+        // handle error
+        abort();
+    }
+
+    if (!timer->Sans)
+    {
+        printf("Error %s", TTF_GetError());
```

Comment:
abort or handle error

Reply:
you can be more specific about what you mean

Reply:
Either call abort() or handle the function appropriately
Just call abort() in the end if you're unsure about how to handle the error